tests/pragma/solidity.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity ^0.4.23;
+pragma   solidity   ^0.4.23;
+
```

Comment:
this should be an example of solidity code?

Reply:
Yep! See my comment below for more details ⬇️